R/categories.R

```diff
@@ -239,3 +239,70 @@ setMethod("lapply", "Categories", function (X, FUN, ...) {
     X@.Data <- lapply(X@.Data, FUN, ...)
     return(X)
 })
+
+#' Change the id of a category for a categorical variable
+#' 
+#' @param variable the variable in a crunch dataset that will be changed
```

Comment:
Note that the variable must be categorical, categorical array, or multiple response.